test/functional/ansible_roles_controller_test.rb

```diff
@@ -2,19 +2,4 @@
 
 require 'test_plugin_helper'
 # functional tests for AnsibleRolesController
-class AnsibleRolesControllerTest < ActionController::TestCase
-  setup do
-    @role = FactoryBot.create(:ansible_role)
-  end
-
-  basic_index_test
-
-  test 'should destroy role' do
-    assert_difference('AnsibleRole.count', -1) do
-      delete :destroy,
-             :params => { :id => @role.id },
-             :session => set_session_user
-    end
-    assert_redirected_to ansible_roles_url
-  end
-end
+class AnsibleRolesControllerTest < ActionController::TestCase; end
```

Comment:
If it's an empty class, is there any point in keeping it at all? I'd also note that there are system tests (https://guides.rubyonrails.org/testing.html#system-testing) that can use a real browser. I'd love to still have some tests.

Reply:
I suppose I could remove it, but I would like to have a test for the `format` action at some point.
I will definitely add tests for the code changed in this PR though, just wanted us to agree on the implementation details first.